tests/saw/spec/handshake/rfc_handshake_tls13.cry

```diff
@@ -149,10 +149,10 @@ doHandshake params = (foldl doState initialHandshake (take`{n} (stateMachine par
  *                      |    +---------------------+
  *                      |    |  HELLO_RETRY_STATE  |
  *         if not retry |    |---------------------|
- *                      |    |helloRetryRequest (S)|
+ *                      |    |serverHello (S)      |
  *                      |    |* CCS (S)            |
- *                      |    |* CCS (C)            |
  *                      |    |clientHello (C)      |
+ *                      |    |* CCS (C)            |
```

Comment:
Are we sure this is correct? From the RFC:

> If not offering early data, the client sends a dummy
      change_cipher_spec record (see the third paragraph of Section 5)
      immediately before its second flight.  This may either be before
      its second ClientHello or before its encrypted handshake flight.

https://tools.ietf.org/html/rfc8446#appendix-D.4